waiter/src/waiter/util/utils.clj

```diff
@@ -27,8 +27,7 @@
             [taoensso.nippy.compression :as compression]
             [waiter.util.date-utils :as du])
   (:import clojure.core.async.impl.channels.ManyToManyChannel
-           clojure.lang.ExceptionInfo
-           clojure.lang.PersistentQueue
+           (clojure.lang ExceptionInfo Named PersistentQueue)
```

Comment:
You can delete `Named` here if you remove the special case below.

Reply:
Done.